include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -84,22 +84,23 @@ enum EOverlayType : byte {
 
 table CartesianImpedenceControlMode {
   /// actual stiffness to set rot:[nm/rad]
-  stiffness:EulerPoseParams;
-
+  stiffness:EulerPose;
```

Comment:
I'm not completely sure of the specific other parameters you are referencing, but it depends on if the information must be provided or not. So to move an arm to a cartesian position, it is not optional to provide as per [MoveArmCartesianServo](https://github.com/ahundt/grl/blob/master/include/grl/flatbuffer/ArmControlState.fbs#L31), but for example to configure stiffness, it need not be provided because it can be initialized with a completely reasonable default from processdata. 

Hopefully that's clear?